released/packages/coq-relation-algebra/coq-relation-algebra.1.7.2/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+name: "coq-relation-algebra"
+synopsis: "Relation Algebra and KAT in Coq"
+maintainer: "Damien Pous <Damien.Pous@ens-lyon.fr>"
+homepage: "http://perso.ens-lyon.fr/damien.pous/ra/"
+license: "LGPL"
+depends: [
+  "ocaml"
+  "coq" {>= "8.10" & < "8.11~"}
+]
+depopts: [ "coq-mathcomp-ssreflect" ]
+build: [
+  ["sh" "-exc" "./configure --%{coq-mathcomp-ssreflect:enable}%-ssr"]
```

Comment:
Does the following work? (not tested)
```suggestion
  ["./configure" "--%{coq-mathcomp-ssreflect:enable}%-ssr"]
```

Reply:
Hum... it seems there are no `configure` file in the archive or Git repository actully.